root/static/scripts/edit/utility/linkPhrase.js

```diff
@@ -27,8 +27,8 @@ const entity0Subst = /\{entity0\}/;
 const entity1Subst = /\{entity1\}/;
 
 export type CachedLinkPhraseData<T> = {
-  attributeValues: ?{+[string]: Array<T> | T, ...},
-  phraseAndExtraAttributes: {[string]: [T, T], ...},
+  attributeValues: ?{+[attributeName: string]: Array<T> | T, ...},
+  phraseAndExtraAttributes: {[key: string]: [T, T], ...},
```

Comment:
We should specify what the key *is*. @mwiencek: you worked with this recently, maybe you can give a hint? :)

Reply:
Hi, i have fixed according to all of your suggestions. Thanks :DD. Relating this particular case, maybe change it to `phraseProp`?  Although it doesn't seem to fully convey the meaning.

Reply:
`phraseKey` should work - it includes the link phrase, but also a flag indicating whether `forGrouping` is enabled.